core/interop/cats-mtl/src/main/scala/tofu/core/interop/catsmtl/implicits.scala

```diff
@@ -0,0 +1,67 @@
+package tofu.core.interop.catsmtl
+
+import cats.mtl.{Ask, Local, Handle => MHandle, Raise => MRaise}
+import cats.syntax.functor._
+import cats.{Applicative, Functor}
+import tofu.{Errors, Raise, WithContext, WithLocal}
+
+object implicits {
+
+  implicit def deriveLocal[F[_]: Applicative, C](implicit L: WithLocal[F, C]): Local[F, C] = new Local[F, C] {
+    override def local[A](fa: F[A])(f: C => C): F[A] = L.local(fa)(f)
```

Comment:
I think that the `override` word is syntax noise when you implement a trait with all abstract members. I would omit it everywhere unless it's _real_ overriding.

Reply:
I'm already done it locally, but now I've stuck with `diverging implicit expansion`.